Revit_Core_Engine/Create/Definition/Parameter.cs

```diff
@@ -49,9 +47,9 @@ public static partial class Create
         [Output("definition", "Revit parameter Definition created based on the input properties.")]
         public static Definition Parameter(Document document, string parameterName, string typeName, string groupName, bool instance, IEnumerable<string> categoryNames, bool shared, string discipline = "")
         {
-#if (REVIT2020 || REVIT2021 || REVIT2022)
+#if REVIT2020 || REVIT2021 || REVIT2022
             List<ParameterType> parameterTypes = new List<ParameterType>();
-            foreach (ParameterType pt in Enum.GetValues(typeof(ParameterType)))
+            foreach (ParameterType pt in System.Enum.GetValues(typeof(ParameterType)))
```

Comment:
why did you remove `using System` and add full qualification? that seems to be causing unnecessary noise...